modin/pandas/test/test_api.py

```diff
@@ -33,11 +33,9 @@ def test_top_level_api_equality():
         "core",
         "compat",
         "util",
-        "offsets",
         "datetime",
         "arrays",
         "api",
-        "tseries",
```

Comment:
Why haven't these modules been added to Modin before?
Should we add the rest of this list? (Given that Modin is used in the same context as Pandas)

Reply:
@devin-petersohn didn't mention why we didn't add them before. From a quick inspection, the rest of this list seems to be pandas internal implementations. I'd rather not try exposing them via `modin.pandas`.

Reply:
ok